src/main/java/uc/course/updatemembers/UpdateCMemDsGateway.java

```diff
@@ -0,0 +1,12 @@
+package uc.course.updatemembers;
+
+import java.util.List;
+
+public interface UpdateCMemDsGateway
+{
+    boolean checkIfCourseExists(String courseId);
+    void addCourseEnrollment(String courseId, String userId);
+    void removeCourseEnrollment(String courseId, String userId);
+    List<String> getCourseEnrollment(String userId);
```

Comment:
method name will be updated to getEnrolmentsByUserId for consistency with other use cases